### PR TITLE
Load .env file in Application::boot

### PR DESCRIPTION
**Pull request by bvisness**
josegonzalez\Dotenv was already in composer.json, but was never actually being used.

### CODE REVIEW

**Comment by shadowhand** [comment]
I would merge the first and third commits into one.

**Comment by bvisness** [comment]
Believe me, I tried...

File: src/Application.php [hunk 1]
```diff
@@ -19,6 +20,10 @@ class Application
      */
     public static function boot(Injector $injector = null)
     {
+        $Loader = new Loader(APP_PATH . '.env');
```
Comment (shadowhand) [inline src/Application.php]: This should be moved after the injector default, and look like this:

```php
$loader = $injector->make('josegonzalez\Dotenv\Loader');
...
```

File: src/Application.php [hunk 2]
```diff
@@ -8,6 +8,7 @@
 use Psr\Http\Message\ServerRequestInterface;
 use Spark\Adr\RouteInterface;
 use Spark\Adr\ActionInterface;
+use josegonzalez\Dotenv\Loader;
```
Comment (shadowhand) [inline src/Application.php]: You can drop this, since the injector contains it.

**Comment by shadowhand** [comment]
@dolfelt @ameech you okay with this change? Or should we be forcing the `.env` loader to be in `web/index.php`?

**Comment by ameech** [comment]
I think its fine in application boot. Especially if we ever add command line support.

**Comment by dolfelt** [comment]
:+1:

**Comment by bvisness** [comment]
I think that should finally do it...go ahead and merge at your leisure.

**Comment by shadowhand** [comment]
Just waiting for Travis to finish. :)

**Pull request merged**